src/test/crimson/test_messenger_thrash.cc

```diff
@@ -107,7 +107,7 @@ class SyntheticDispatcher final
   std::optional<seastar::future<>> ms_dispatch(crimson::net::ConnectionRef con,
                                                MessageRef m) {
     if (verbose) {
-      logger().warn("{}: con = {}", __func__, con);
+      logger().warn("{}: con = {}", __func__, fmt::ptr(con.get()));
```

Comment:
`Connection` is formattable, so `*con` is simpler instead of `fmt::ptr()`.

Same applies to the below 4 places.

Reply:
fixed and repushed